src/adapters/suggest.js

```diff
@@ -95,23 +95,14 @@ export default class Suggest {
 
       onSelect: (e, term, item, items) => {
         e.preventDefault()
-        const type = item.getAttribute('data-type')
-        const val = item.getAttribute('data-val')
         const itemId = item.getAttribute('data-id')
 
-        if ('category' === type) {
-          PanelManager.openCategory({
-            category: CategoryService.getCategoryByName(val)
-          })
-        }
-        else {
-          let prefixPoint = this.prefixes.find((prefix) => prefix.id === itemId)
-          if(prefixPoint) {
-            this.onSelect(prefixPoint)
-          } else {
-            let poi = items.find(poi => poi.id === itemId)
-            this.onSelect(poi)
-          }
+        let prefixPoint = this.prefixes.find((prefix) => prefix.id === itemId)
+        if(prefixPoint) {
+          this.onSelect(prefixPoint)
+        } else {
+          let selectedItem = items.find(item => item.id === itemId)
```

Comment:
Don't you check if you found something in here?

Reply:
Or if `items` actually exists?

Reply:
a default value for `items` has been added as a precaution, thanks